src/main/java/org/opentripplanner/standalone/config/RoutingRequestMapper.java

```diff
@@ -160,4 +162,12 @@ private static void mapTransferOptimization(TransferOptimizationRequest p, NodeA
     p.extraStopBoardAlightCostsFactor =
       c.asDouble("extraStopBoardAlightCostsFactor", p.extraStopBoardAlightCostsFactor);
   }
+
+  private static void mapUnpreferred(
```

Comment:
Does it make sense to have a method for a single line of code that is never repeated?

Reply:
Yes, this doesn't make sense.